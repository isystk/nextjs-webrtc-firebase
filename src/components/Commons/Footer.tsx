import React, { FC } from 'react'

export const CommonFooter: FC = () => {
  return (
    <>
      <footer className="App-footer">
        <strong>
          Copyright &copy; 2021-2022{' '}
          <a href="#" className="App-link">
            isystk.com
          </a>
          .
        </strong>{' '}
        All rights reserved.
      </footer>
    </>
  )
}

export default CommonFooter
