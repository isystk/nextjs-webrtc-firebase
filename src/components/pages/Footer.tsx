import React, { VFC } from 'react'

const CommonFooter: VFC = () => {
  return (
    <>
      <footer className="App-footer">
        <strong>
          Copyright &copy; 2021-2022{' '}
          <a href="#" className="App-link">
            isystk.com
          </a>
          .
        </strong>{' '}
        All rights reserved.
      </footer>
    </>
  )
}

export default CommonFooter
