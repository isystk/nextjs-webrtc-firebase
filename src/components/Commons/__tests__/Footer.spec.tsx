import React from 'react'
import renderer from 'react-test-renderer'
import { CommonFooter } from '../Footer'

test('Footer', () => {
  const component = renderer.create(<CommonFooter />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
