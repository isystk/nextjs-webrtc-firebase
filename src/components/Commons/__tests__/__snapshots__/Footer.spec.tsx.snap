// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<footer
  className="main-footer"
>
  <strong>
    Copyright Â© 2019-2020 
    <a
      href="#"
    >
      isystk.com
    </a>
    .
  </strong>
   
  All rights reserved.
</footer>
`;
